build.gradle.kts

```
@@ -16,10 +17,18 @@ repositories {
 }
 
 dependencies {
-	implementation("org.springframework.boot:spring-boot-starter")
+	implementation("org.springframework.boot:spring-boot-starter:2.5.6")
```

я бы советовал выносить общие версии зависиомтсей в переменные. 
```suggestion
        val spring = "2.5.6"
	implementation("org.springframework.boot:spring-boot-starter:$spring")
```